### PR TITLE
1_0_15

### PR DESCRIPTION
**Pull request by daveboutcher**
pyocient 1.0.15 release notes

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 # Pyocient Release Notes
 
+## 1.0.15
+
+- The dependency on protobuf has been upgraded from `>=3.12.0,<=3.19.1` to `>=3.20.0,<=4.22.0`
+- Fixed an error when no host parameter is specified on the `connect()` api.
 ## 1.0.14
 - Updated pyocient dependencies to eliminate some unused dependencies and tighten
   the dependencies on protobuf and dsnparse
```
Comment (rmasters1) [inline README.md:9]:
```suggestion
  the dependencies on protobuf and dsnparse.
```

File: README.md [hunk 2]
```diff
@@ -1,8 +1,12 @@
 # Pyocient Release Notes
 
+## 1.0.15
+
+- The dependency on protobuf has been upgraded from `>=3.12.0,<=3.19.1` to `>=3.20.0,<=4.22.0`
```
Comment (rmasters1) [inline README.md:5]:
```suggestion
- The dependency on protobuf has been upgraded from `>=3.12.0,<=3.19.1` to `>=3.20.0,<=4.22.0`.
```

**Review by rmasters1 — APPROVED** [review]
After slight edits to wording and addition of periods, the release notes look good.

Processed via suggested changes that I committed for efficiency.

**Comment by daveboutcher** [comment]
Thanks @rmasters1

**Pull request merged**